### DISCUSSION TITLE
[blocks]: charts

### DISCUSSION THREAD

**Discussion by ayushpathak-48** [shadcn-ui/ui#3177] [Blocks Request]
### Description

A dashboard with statistics and charts

**Comment by ashishkumarbhati** [comment]
I too need this very badly.

**Comment by ruymon** [comment]
Agree! TBH I think we should extend this discussion to implement a charts component....

**Comment by ezeparziale** [comment]
What libraries are you using for charts?

**Reply by ruymon** [reply]
@ezeparziale In my current projects I am using `apexcharts` but I've seen other people use Recharts too...

**Reply by ezeparziale** [reply]
I tested `tremor` but I had problems with theme config.

**Comment by arthurdotwork** [comment]
Looking at the Dashboard example on https://ui.shadcn.com, it seems like it is using Recharts, maybe it could just be an article about how to style Recharts graph properly so it matches, that could represent less effort than building a chart library.

![CleanShot 2024-03-26 at 14 49 27](https://github.com/shadcn-ui/ui/assets/35616365/196d8348-7b84-40da-b32f-52eca3a7cb52)

**Reply by ruymon** [reply]
Well... I still think a wrapper component for recharts would be great....

**Reply by ajayvignesh01** [reply]
I think that's pretty much what tremor is, minus the shadcn styling

**Comment by shadcn** [comment]
Shipped! https://ui.shadcn.com/charts 

Thanks everyone.
